### PR TITLE
Basic CNN Architecture WIP(Not Ready for Merge)

### LINKED ISSUE #1: Create Basic CNN Architecture
As mentioned in our last meeting, get a basic CV classification CNN working.

### PR DESCRIPTION
**Pull request by abhon**
I started on the Basic CNN Architecture: 2 layers, and has the parameters to determine how large you want each channel to be etc., dropout probability, activation function. I am still working on the forward, but any Feedback is appreciated(extra parameters, clean it up, etc.)  I've set Alex to review it but anyone can review!

### CODE REVIEW

File: Basic_CNN_Architecture [hunk 1]
```diff
@@ -1,19 +1,45 @@
-import torch
-import torch.nn as nn
```
Comment (alexwdong) [inline Basic_CNN_Architecture]: Did you remove the import statements by accident?

Comment (abhon) [reply Basic_CNN_Architecture]: Yeah, my bad. Was trying things out in jupyter and had the imports in another cell. Will add it back after I work on it some more.

File: Basic_CNN_Architecture.py [hunk 2]
```diff
@@ -0,0 +1,48 @@
+import torch
+import torch.nn as nn
+
+class BasicCNN(nn.Module):
+	def __init__(self, 
+			in_channels: int, 
+			enc_channels: int,
```
Comment (alexwdong) [inline Basic_CNN_Architecture.py:7]: _enc_channels_ (input line 7) seems to be used both in Convolution layers(line 18,23) and in the Linear layers (31,32)

I think _enc_channels_ (line 18,23) represents number of output convolution channels for the first Convolution layer, out channels represents number of output convolutional layers after the second layer

I think we should have two separate variables for the number intermediate channels in between the convolutional layers, and the number of intermediate nodes between the two linear layers.

File: Basic_CNN_Architecture.py [hunk 3]
```diff
@@ -0,0 +1,48 @@
...
+					in_channels = enc_channels, 
+					out_channels = out_channels,
+					kernel_size = kernel_size, 
+					stride = stride
+					)
+		self.pool1 = nn.MaxPool2d(enc_channels)
+		self.pool2 = nn.MaxPool2d(out_channels)
+
+		self.fc1 = nn.Linear(out_channels, enc_channels)
+		self.fc2 = nn.Linear(enc_channels, num_classes)
```
Comment (alexwdong) [inline Basic_CNN_Architecture.py]: Should there be relu's here?

File: Basic_CNN_Architecture.py (hunk 2 above)
Comment (abhon) [reply Basic_CNN_Architecture.py:7]: Yeah I can make another parameter for that.

File: Basic_CNN_Architecture.py (hunk 3 above)
Comment (abhon) [reply Basic_CNN_Architecture.py]: Will make a parameter so we can choose what activation function we want. Do you also want batch normalization?
